.github/workflows/reusable-e2e.yml

```diff
@@ -327,17 +331,17 @@ jobs:
       - name: Checkout Amplify UI
         uses: actions/checkout@24cb9080177205b6e8c946b17badbe402adc938f # v3.4.0 https://github.com/actions/checkout/commit/24cb9080177205b6e8c946b17badbe402adc938f
         with:
-          ref: ${{ inputs.commit }}
+          ref: ${{ inputs.pr-branch }}
           repository: ${{ inputs.repository }}
-          fetch-depth: 2
+          fetch-depth: 0
```

Comment:
For my knowledge, why this gets updated to 2?

Reply:
We need to fetch all history bc we are looking at all commits on a branch not just the latest.